runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -156,6 +159,8 @@ static ALL_COSTS: &[(Cost, fn(&mut EstimatorContext) -> GasCost)] = &[
     (Cost::TouchingTrieNode, touching_trie_node),
     (Cost::GasMeteringBase, gas_metering_base),
     (Cost::GasMeteringOp, gas_metering_op),
+    (Cost::RocksDbInsertValueByte, rocks_db_insert_value_byte),
+    (Cost::RocksDbReadValueByte, rocks_db_read_value_byte),
```

Comment:
```suggestion
    (Cost::RocksDbInsertValueByte, rocks::db_insert_value_byte),
    (Cost::RocksDbReadValueByte, rocks::db_read_value_byte),
```

For "tablish, rectangular" code, qualified imports usually work better.

Reply:
These are actually two local functions that delegate to the functions in RocksDB.
I feel the only reason to keep the indirection is that the inner function can use the `Config` but not `EstimatorContext`, which makes it very clear that it is separate from the usual testbed and DB.
But no strong opinion on my side. I can also remove the indirection.

Reply:
Ah, that makes total sense to me know, yeah, this is indeed better.